packages/api-v2/src/hooks/useAvailableCTraderAccounts.ts

```diff
@@ -0,0 +1,42 @@
+import { useMemo } from 'react';
+import useQuery from '../useQuery';
+
+const market_type_to_leverage_mapper: Record<string, number> = {
+    gaming: 500,
+    financial: 1000,
+    all: 100,
```

Comment:
hey @hasan-deriv 
just curious, do we have leverages in our API, not fully happy to hard code them here

Reply:
> do we have leverages in our API

yes in an upcoming BE card [CFDS-3412](https://app.clickup.com/t/20696747/CFDS-3412), but it shouldn't become a blocker since the rest of the high code is referring to some kind of cfd-config